### ISSUE TITLE
Unable to install Vienna RSS (macOS 10.15.6)

### ISSUE THREAD

**Issue by gercrypt0** [ViennaRSS/vienna-rss#1315]
<img width="764" alt="ViennaRss error" src="https://user-images.githubusercontent.com/69248999/89416180-26a3f180-d72d-11ea-866a-325a933de6c6.png">

After I downloaded Vienna RSS (3.5.5) and tried to instal it the attached notice came up. 
It says: "Vienna can't be installed because Apple can't search it for malware. Ask the developer for further information".

**Comment by Eitot** [comment]
This appears to be a notarisation issue. The app itself is properly code-signed though.
```bash
% spctl --assess -vvv ~/Downloads/Vienna.app 
/Users/<user>/Downloads/Vienna.app: accepted
source=Developer ID
origin=Developer ID Application: Barijaona Ramaholimihaso (KUU2LM7U9K)
```

As a workaround, you can open it either right-clicking on it in Finder or by going to System Preferences:
<img width="780" alt="Screenshot 2020-08-07 at 06 15 55" src="https://user-images.githubusercontent.com/15073177/89608454-75f53980-d875-11ea-8a76-c4564bd33e84.png”>

@barijaona Did you upload a notarised copy?
```bash
% stapler validate ~/Downloads/Vienna.app      
Processing: /Users/<user>/Downloads/Vienna.app
Vienna.app does not have a ticket stapled to it.
```

**Comment by josh64x2** [comment]
@Eitot I don't think we've released a notarised version yet. It will be in our next release.

**Comment by barijaona** [comment]
A notarized version, 3.5.6, will be released as soon as PR #1316 and #1317 are reviewed.

**Comment by barijaona** [comment]
Fixed by version 3.5.6 (notarised build)
